xtask/Cargo.toml

```diff
@@ -24,5 +26,5 @@ sha2 = "0.10.6"
 syn = { version = "2.0.0", features = ["full"] }
 tar = "0.4.38"
 tempfile = "3.6.0"
+ureq = { version = "2.8.0", features = ["http-interop"] }
```

Comment:
nit: any reason why you prefer `ureq` over `hyper` here?

Reply:
No super strong reason, but generally I agree with this section of the ureq docs: https://docs.rs/ureq/2.8.0/ureq/#blocking-io-for-simplicity

Basically, async IO (which hyper is built around) doesn't add much for a very low-traffic client like this, and would also significantly expand the dep tree.

Reply:
Got it. I don't have a strong opinion here either.